kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -120,6 +126,35 @@ public SkeletonWindowController(EditorController editorController, String docume
         if (editorController.getFxomDocument() != null) {
             editorController.getFxomDocument().sceneGraphRevisionProperty().addListener(fxomDocumentRevisionListener);
         }
+        
+        this.copyAccelerator = getCopyToClipboardKeyAccelerator();
+    }
+
+    private KeyCharacterCombination getCopyToClipboardKeyAccelerator() {
+        if (EditorPlatform.IS_MAC) {
+            return new KeyCharacterCombination("c", KeyCombination.META_DOWN);
+        }
+        return new KeyCharacterCombination("c", KeyCombination.CONTROL_DOWN);
+    }
+
+    private EventHandler<KeyEvent> getCopyKeyEventHandler() {
+        if (EditorPlatform.IS_MAC) {
+            return event -> handleCopyToClipboardEvent(event, "c".equals(event.getText()), event.isMetaDown());
+        } else {
+            return event -> handleCopyToClipboardEvent(event, KeyCode.C.equals(event.getCode()), event.isControlDown());
+        }
```

Comment:
use `isShortcutDown` instead

Reply:
Same here, if it works well using `isShortcutDown`, this method can possibly ger removed. Its only there due to the conditional.